### DISCUSSION TITLE
Why my University Email are not allowed?

### DISCUSSION THREAD

**Discussion by akbarsaputrait** [community/community#53741] [GitHub Education]
Since 2020 i was able to use GitHub Developer Student Benefits until today. I got an email about expiry of my GitHub Academic coupon. When i want to extend the period, it's say "We do not allow application from this email domain". Please help.. Thank you
![image](https://user-images.githubusercontent.com/27531592/234453039-e706d2d1-f019-4016-a8b6-d1f7cdc72286.png)

**Comment by samus-aran** [comment]
Hi @akbarsaputrait 

**"Why can't I use this ?"** expands to display the following:

> We do not allow applications from this email domain. Please apply with a different email address.

**Reply by ridhaagam** [reply]
![image](https://user-images.githubusercontent.com/71159151/235163607-5539f6e5-18e1-48a7-a6e7-b6bdff4b8614.png)

It gave that answer above, I also had this difficulty and it is listed as a fraud domain. But, when adding "What is the name of your school?" the domain was listed there.

**Reply by akbarsaputrait** [reply]
![image](https://user-images.githubusercontent.com/27531592/235171852-b5683fea-0844-4bc1-9417-048de80d28dd.png)

seems like that? @ridhaagam

**Reply by ridhaagam** [reply]
Yes like that..

**Reply by akbarsaputrait** [reply]
> ![image](https://user-images.githubusercontent.com/27531592/235171852-b5683fea-0844-4bc1-9417-048de80d28dd.png)
> 
> seems like that? @ridhaagam 

Any solutions for this? @samus-aran
